zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/HDFSNotebookRepo.java

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright 2015 NAVER Corp. All rights Reserved.
```

Comment:
Please make sure all the code is distributed under one of the [ASF category-a](https://www.apache.org/legal/resolved#category-a) licenses